README.md

```diff
@@ -1,81 +1,113 @@
-[networking options](#networking-options), described below.  Each of the demonstrations indicates where the instructions
-differ for each of these networking options.
+See the [Networking options](#networking-options) below for more details on 
+each of these different networking options.
 
-We also provide a demonstration of a [Docker Swarm cluster](docs/getting-started/powerstrip/CalicoSwarm.md) networked 
-using Calico. This particular demonstration utilizes the [powerstrip](#docker-with-powerstrip) networking option.
+With each of these tutorials we provide details for running the demonstration 
+using manual setup on your own servers, or with a quick set-up in a virtualized
+environment using Vagrant, or a number of cloud services.
 
-For a quick start, we'd recommend following one of the Vagrant installations using the default docker networking.
+We also provide a demonstration of a 
+[Docker Swarm cluster](docs/getting-started/powerstrip/CalicoSwarm.md) 
+networked using Calico. This particular demonstration utilizes the 
+[powerstrip](#docker-with-powerstrip) networking option.
 
 
 ### Networking options
 
-#### Default docker networking
+#### Docker default networking
 
-This uses Dockers standard networking infrastructure, requiring you to explicitly add a created contained into a
-Calico network.
+This uses Dockers standard networking infrastructure, requiring you to 
+explicitly add a created container into a Calico network.
+
+This is compatible with all Docker versions from 1.6 onwards.
 
 #### Docker with libnetwork
 
-Docker's native [libnetwork network driver](https://github.com/docker/libnetwork) is available in the Docker
-[experimental channel](https://github.com/docker/docker/tree/master/experimental) alongside the Docker 1.7 release.
-Docker's experimental channel is still moving fast and some of its features are not yet fully stable.  The current
-master branch of calico-docker is largely focussed on interworking with libnetwork.
+Docker's native [libnetwork network driver][libnetwork] is available in the 
+Docker [experimental channel][docker-experimental] alongside the Docker 1.7 
+release. Docker's experimental channel is still moving fast and some of its 
+features are not yet fully stable.
+
+Setup of the libnetwork environment is a little more involved since it requires
+the experimental Docker version, and a separate datastore used for Docker 
+clustering.
 
 #### Docker with Powerstrip
 
-Powerstrip is a pluggable HTTP proxy for the Docker API.  Development of the Powerstrip version of Calico is lagging 
-behind the master branch, so an older version of `calicoctl` and the `calico-node` docker image are required.
+[Powerstrip][powerstrip] is a pluggable HTTP proxy for the Docker API.  
+Development of the Powerstrip version of Calico is lagging behind the master 
+branch, so an older version of `calicoctl` and the `calico-node` docker image 
+are required.
   
-
 ## How does it work?

```

Comment:
I still don't love this readme. Maybe it's good enough, lets see what Alex thinks but maybe we should have a call to work out how it could be made better.